google/logging/v2/logging_gapic.yaml

```diff
@@ -6,6 +6,7 @@ language_settings:
     package_name: google.cloud.gapic.logging.v2
   go:
     package_name: cloud.google.com/go/logging/apiv2
+    domain_layer_location: cloud.google.com/go/logging
```

Comment:
We split the handwritten logging client into two packages: logging, for writing logs, and logging/logadmin, for reading logs and the other stuff.